win32/src/threadpool_win32.c

```diff
@@ -625,6 +627,14 @@ int threadpool_timer_restart(TIMER_INSTANCE_HANDLE timer, uint32_t start_delay_m
             timer, start_delay_ms, timer_period_ms);
         result = MU_FAILURE;
     }
+    else if(
+        /* Codes_SRS_THREADPOOL_WIN32_07_001: [ If timer state is in destroying state, threadpool_timer_restart shall fail and return a non - zero value. ]*/
+        interlocked_add(&timer->is_destroying, 0) == 1
+        )
+    {
+        LogError("timer is in closing state");

```

Comment:
>closing [](http://example.com/codeflow?start=30&length=7)

I thought the states are CREATED/DESTROYING/DESTROYED.
A wild closing appears :-)

Reply:
modified